admin/class-pw-admin-admin.php

```
@@ -0,0 +1,1009 @@
+            'tmp_name' => $tmp
+        );
+
+        $id = media_handle_sideload($file_array, $post_id);
+
+        if (is_wp_error($id)) {
+            @unlink($file_array['tmp_name']);
+            return false;
+        }
+
+        return $id;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'create_attachment_from_url'. (undefined)

(UnusedPrivateMethod)